drivers/periph_common/Kconfig

```diff
@@ -125,6 +125,8 @@ config MODULE_PERIPH_INIT_RTC
     default y if MODULE_PERIPH_INIT
     depends on MODULE_PERIPH_RTC
 
+osource "$(RIOTCPU)/$(CPU)/periph/Kconfig.rtc"
```

Comment:
can we be consistent here, the rest of the file uses `rsource "Kconfig.<periph>"`

Reply:
Moved the RTC-related symbols to its own file